movement/validators/datasets.py

```diff
@@ -176,3 +174,139 @@ def __attrs_post_init__(self):
                 "Keypoint names were not provided. "
                 f"Setting to {self.keypoint_names}."
             )
+
+
+@define(kw_only=True)
+class ValidBboxesDataset:
+    """Class for validating bounding boxes' data for a ``movement`` dataset.
+
+    We consider 2D bounding boxes only.
+
+    Attributes
+    ----------
+    position_array : np.ndarray
+        Array of shape (n_frames, n_individual_names, n_space)
+        containing the bounding boxes' centroid positions. It will be
+        converted to an `xarray.DataArray` object named "position".
```

Comment:
I'd remove the last sentence about it being converted to xarray. That's what the `from_file()` or `from_numpy()` funcs will do (or their equivalents for bboxes), but the validator itself doesn't do any conversions to xarray, so slightly confusing to mention that here. Ditto for the confidence and shape arrays.